roles/contiv_network/tasks/main.yml

```diff
@@ -21,33 +36,33 @@
   copy: src=netmaster.service dest=/etc/systemd/system/netmaster.service
   when: run_as == "master"
 
-- name: start netmaster
-  shell: systemctl daemon-reload && systemctl start netmaster
+- name: restart netmaster
```

Comment:
![Image of Yaktocat](https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTk8-cmsVhdn6dk3nmGr_exs74GOB9OL_-AI09E0KcpqZpw-pS6kw)


Also it's not a big change. But I can accomodate this if you would like to keep it this way.